editor/editor_properties_array_dict.h

```diff
@@ -114,6 +114,9 @@ class EditorPropertyArray : public EditorProperty {
 	PropertyHint subtype_hint;
 	String subtype_hint_string;
 	LocalVector<Slot> slots;
+	NodePath base_hint;
```

Comment:
This doesn't work, as this is never set, you need to adjust it to fit this class

Reply:
Is this even needed in this case ? I fail to see the purpose here. Could it just be scrapped ?

Reply:
I don't know, I'm not familiar with the code, you should look at how it works and what you need